<?xml version="1.0" encoding="utf-8"?>
<report>
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="38f2fa16f14fe1a"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="pandey"
		host="FLR1622"
		rxversion="5.4.5.19886"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="3200x1080"
		timestamp="4/15/2016 11:02:48 AM"
		timestampiso="2016-04-15T09:02:48"
		timeoutfactor="1"
		module="LocatingFolder">
		<detail>
			LocatingFolder
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:12.435"
			timeRelativeToTestModuleStartTime="00:12.435"
			timeWallClock="11:03:01"
			time="00:12.435"
			level="Error"
			category="User"
			errimg=".\LocatingFolder.cs_20160415_110245_e0a89623_trace.jpg"
			errthumb=".\LocatingFolder.cs_20160415_110245_e0a89623_trace_thumb.jpg">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
